src/fsharp/TastPickle.fs

```diff
@@ -1268,6 +1271,13 @@ let p_tys = (p_list p_ty)
 
 let fill_p_attribs,p_attribs = p_hole()
 
+// In F# 4.5, the type of the "this" pointer for structs is considered to be inref for the purposes of checking the implementation
+// of the struct.  However for backwards compat reaons we can't serialize this as the type.
+let checkForInRefStructThisArg st ty = 
+    let g = st.oglobals
```

Comment:
What is `st.oglobals`?

Reply:
Just a `TcGlobals` like `g` elsewhere in the compiler.